src/views/commentsViews/commentsCard.js

```diff
@@ -18,9 +18,15 @@ import Moment from "react-moment";
 const CommentCard = props => (
   <div className="comment">
     <ViewCard {...props} />
-    <Button outline color="primary" id={`reply${props.id}`}>
+    <Button  color="primary" id={`reply${props.id}`}className="replies-btn">
```

Comment:
This button could also look similar to the Likes button, that is, without the borders of the button, like a mere text, Please make that change to the implementation.

Reply:
@LehruAngel we have changed it to text.